source/declarations.tex

```diff
@@ -1923,25 +1923,28 @@
 are determined as follows:
```

Comment:
On the line above, do we want `E` to be a corresponding *initializer-clause* rather than a corresponding initializer?

Reply:
I guess so: An assignment-expression, for example, is an initializer-clause.